apps/web/public/static/locales/en/common.json

```
@@ -3742,5 +3745,19 @@
   "voice_id": "Voice ID",
   "use_voice": "Use Voice",
   "current_voice": "Current Voice",
+  "setup_incoming_agent": "Set up incoming agent",
+  "setup_incoming_agent_description": "Configure an AI agent to handle incoming calls on your phone number",
+  "connect_a_phone_number_first": "Connect a Phone Number first",
+  "setup_agent_for_incoming_calls": "Set up Agent for incoming calls",
+  "configure_agent_to_handle_incoming_calls": "Configure agent to handle incoming calls",
+  "incoming_calls": "Incoming Calls",
+  "outgoing_calls": "Outgoing Calls",
+  "inbound_agent_setup_success": "Inbound agent set up successful",
+  "inbound_agent_configured": "Inbound agent configured",
+  "setup_inbound_agent": "Set up Inbound Agent",
+  "please_select_event_type_first": "Please select an event type first",
+  "edit_configuration": "Edit Configuration",
+  "select_event_type_for_inbound_calls": "Inbound calls can book only one event type. Select the event type where meetings will be scheduled when callers reach your agent.",
+  "setup_inbound_agent_for_incoming_calls": "Setup inbound agent for incoming calls",
```

```suggestion
  "setup_inbound_agent_for_incoming_calls": "Set up inbound agent for incoming calls",
```
And I think that should be like that because it's a verb